spec/views/list_has_20_items_spec.rb

```diff
@@ -0,0 +1,42 @@
+feature "List only has twenty items per page" do
+  scenario "posts list has 20 posts per page" do
+    me = Fabricate(:user)
+    30.times do
+      Fabricate(:post, author: me)
+    end
+    signin_as me
+    render_template layout: "users/index"
```

Comment:
lines 3-6 were to sign in as a user and to make posts attributed to that user. I needed that to test if items in a list. then the render_template layout was to see if I was on the correct page. I was having issues with assert_template and the docs I found said that render would do the same thing.